tools/bqpipeline/.gitignore

```diff
@@ -0,0 +1,8 @@
+.idea
```

Comment:
This doesn't have a * because the others need to match any file with those extensions while this only needs to exclude the '.idea' directory generated by PyCharm